### ISSUE TITLE
No visual focus indicator on elements in the Table of Contents.

### ISSUE THREAD

**Issue by sdellis** [pulibrary/pulfalight#494]
The focused element (in this case, the `<i>` or the `<li>` elements) should have a visible outline around it when in focus, at least when navigating via keyboard. This can be done by setting the focus element state via css.  

Severity: Critical

WCAG Guideline: 
2.4.7 AA: The page element with the current keyboard focus has a visible focus indicator.
